src/streamlink/plugins/aliez.py

```diff
@@ -8,13 +8,15 @@
 from streamlink.stream import HTTPStream, RTMPStream
 
 _url_re = re.compile(r"""
-    http(s)?://(\w+\.)?aliez.tv
+    http(s)?://(\w+\.)?aliez.\w+/
```

Comment:
You could do the regex like this:
```python
_url_re = re.compile(r"https?://(?:\w+\.)?aliez(?:\.\w+)+/(?:live|video/\d+)/.+")
```
There's no need for so many groups with the live/video part of the URL. If you make the domain part repeatable it can support `.tv`, `.co.me`, etc. They don't really have to be non-capturing groups either :)